### PR TITLE
Updating test descriptions

### PR DESCRIPTION
**Pull request by mlunoe**
Follow up PR to: https://github.com/dcos/dcos-ui/pull/1584

This PR fixes test descriptions to better describe expected results

### CODE REVIEW

File: plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js [hunk 1]
```diff
@@ -20,19 +21,23 @@ describe('MultiContainerNetwork', function () {
         .toEqual({mode: 'host'});
     });
 
-    it('should return a container object', function () {
+    it('should return a container with mode container', function () {
```
Comment (orlandohohmeier) [inline plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js]: It's not returning a container, is it? How about a "config" or "object"?

File: plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js [hunk 2]
```diff
@@ -20,19 +21,23 @@ describe('MultiContainerNetwork', function () {
...
+      batch = batch.add(
+        new Transaction(['network'], `${Networking.type.CONTAINER}.foo`)
+      );
 
       expect(batch.reduce(MultiContainerNetwork.JSONReducer.bind({}), {}))
       .toEqual({mode: 'container', name: 'foo'});
     });
 
-    it('should return a container object', function () {
+    it('should return a container with mode host', function () {
```
Comment (orlandohohmeier) [inline plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js]: Same here.

**Comment by orlandohohmeier** [comment]
Thanks a lot for taking care. A few minor nits but the rest looks great!

File: plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js (hunk 1 above)
Comment (mlunoe) [inline plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js]: You are right!

File: plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js (hunk 2 above)
Comment (mlunoe) [inline plugins/services/src/js/reducers/serviceForm/__tests__/MultiContainerNetwork-test.js]: Updated!

**Pull request merged**